src/controller/java/BUILD.gn

```diff
@@ -351,6 +351,40 @@ kotlin_library("chipcluster_test") {
   kotlinc_flags = [ "-Xlint:deprecation" ]
 }
 
+kotlin_library("matter_controller") {
+  output_name = "MatterController.jar"
```

Comment:
rename it to matter_kotlin_controller? to differentiate the underneath one

Reply:
later we would upload all jar to maven, I am expecting clear differentiation between ChipController.jar and MatterKotlinController.jar so that user could easily get to know what it is..